core/domain/app_feedback_report_services_test.py

```diff
@@ -132,7 +132,7 @@ class AppFeedbackReportServicesUnitTests(test_utils.GenericTestBase):
         },
         'user_supplied_feedback': {
             'report_type': 'suggestion',
-            'category': 'language_suggestion',
+            'category': 'LANGUAGE_SUGGESTION',
```

Comment:
No this is wrong, this needs to `'language_suggestion'` and you need to modify the non-test code so that the tests still pass.

Reply:
Hi @vojtechjelinek, i changed the value since the tests were failing previously as `get_category_from_string` in `app_feedback_report_domain.py` was raising an exception.
should i still change it ?